modules/client-configuration/pages/clients-ubuntu.adoc

```diff
@@ -46,36 +46,98 @@ Some actions are not yet supported:
 
 This section describes how to manually prepare {ubuntu} clients for registration to your {susemgr} Server and synchronize the repositories.
 
-////
-.Procedure: Preparing an {Ubuntu} 16.04 Client for Registration
+// SUSE Manager specific instructions
+//ifeval::[{suma-content} == true]
 
+.Procedure: Setting up the software channels for {ubuntu} Client registration
+
+Ensure you have the corresponding {ubuntu} product enabled and required channels have been fully synchronized:
+
+.Ubuntu 18.04
+* Product: {ubuntu} Client 18.04
+* Mandatory channels: [systemitem]``ubuntu-18.04-pool-amd64``
++
+.Ubuntu 16.04
```

Comment:
Check: "View File" this headline is not there. Some formating problem somewhere ?